src/MainWindow.vala

```diff
@@ -20,6 +20,47 @@
 
 public class PantheonCalculator.MainWindow : Gtk.ApplicationWindow {
     private static GLib.Settings settings;
+    private Gdk.Clipboard clipboard;
+    private Gtk.EventControllerKey event_controller;
+
+    private Button button_0;
+    private Button button_1;
+    private Button button_2;
+    private Button button_3;
+    private Button button_4;
+    private Button button_5;
+    private Button button_6;
+    private Button button_7;
+    private Button button_8;
+    private Button button_9;
+    private Button button_add;
+    private Button button_sub;
+    private Button button_mult;
+    private Button button_div;
+    private Button button_point;
+    private Button button_percent;
+    private Button button_clr;
+    private Button button_par_left;
+    private Button button_par_right;
+    private Button button_pow;
+    private Button button_sr;
+    private Button button_sin;
+    private Button button_sinh;
+    private Button button_cos;
+    private Button button_cosh;
+    private Button button_tan;
+    private Button button_tanh;
+    private Button button_pi;
+    private Button button_e;
+    private Button button_log;
+    private Button button_ln;
+    private Button button_asin;
+    private Button button_acos;
+    private Button button_atan;
+    private Button button_reciprocal;
+    private Button button_m_add;
+    private Button button_m_sub;
+    private Button button_ms;
```

Comment:
Exposing this buttons like this isn't necessary. These all use GLib.Action, so instead of activating the button you can activate the action directly if that's necessary

Reply:
IK, unless you want animated the button to simulate feedback

Reply:
I just closed #220 as "Not planned". I don't think it's worth adding this kind of complexity and fragility for this visual effect

Reply:
> I just closed #220 as "Not planned". I don't think it's worth adding this kind of complexity and fragility for this visual effect

I disagree